earth_enterprise/src/installer/install_server.sh

```diff
@@ -109,10 +218,322 @@ main_postinstall()
      python ./set_geecheck_config.py
   fi
 
-  # 10)
+  # 10) 
   show_final_success_message
 }
 
+#-----------------------------------------------------------------
+# Pre-install Functions
+#-----------------------------------------------------------------
+
+show_intro()
+{	
+	echo -e "\nWelcome to the $GEES $LONG_VERSION installer."
+}
+
+# TODO: Update for GEE Server
+show_help()
+{
+	echo -e "\nUsage: \tsudo ./install_fusion.sh [-dir /tmp/fusion_os_install -ar /gevol/assets -sv /gevol/src -u fusionuser"
+	echo -e "\t\t-g gegroup -nobk -nop -hnf -nostart]\n"
+
+	echo -e "-h \t\tHelp - display this help screen"
+	echo -e "-dir \t\tTemp Install Directory - specify the temporary install directory. Default is [$TMPINSTALLDIR]."	
+	echo -e "-u \t\tFusion User Name - the user name to use for Fusion. Default is [$GEFUSIONUSER_NAME]. \n\t\tNote: this is only used for new installations."
```

Comment:
Correct -- this should be updated to take in values for:  $GEAPACHEUSER_NAME, $GEPGUSER_NAME and $GEPGUSER_NAME